server/src/main/java/org/elasticsearch/index/IndexVersion.java

```diff
@@ -185,8 +183,10 @@ private static IndexVersion registerIndexVersion(int id, Version luceneVersion,
      * READ THE JAVADOC ABOVE BEFORE ADDING NEW INDEX VERSIONS
      * Detached index versions added below here.
```

Comment:
Suggest putting the instructions at the bottom of this block rather than in the javadoc, similar to #98078.

Reply:
Done